index.js

```diff
@@ -112,6 +113,11 @@ gulpPrefixer = function (AWS) {
                     return callback(new gutil.PluginError(PLUGIN_NAME, "S3 headObject Error: " + getErr.stack));
                 }
 
+                if(getData && getData.ETag === '"' + md5(file.contents) + '"') {
```

Comment:
`getData.ETag` is a string that contains quotes. So here I have had to wrap the output of `md5(file.contents)` in `"`